src/main/java/com/autotune/utils/AutotuneConstants.java

```diff
@@ -0,0 +1,12 @@
+package com.autotune.utils;
```

Comment:
Missing legal header here

Reply:
Thanks @dinogun for the review. Will be adding it and updating you.

Reply:
Added legal header